CHANGELOG.md

```diff
@@ -1,4 +1,20 @@
-# Changelog 
+<!--!
```

Comment:
It would be good to add some documentation for the purpose of these comments in the README. It will help us with maintenance going forward. I currently do not know when a doxygen directive is needed, for example.

Reply:
This is done in 
- https://github.com/p4lang/p4c/pull/4875